src/commands.ts

```diff
@@ -650,6 +651,42 @@ const findListsAtSameLevel = (doc: ManuscriptNode, list: NodeWithPos) => {
   return lists
 }
 
+function splitListItem(
+  state: EditorState,
+  dispatch: (tr: ManuscriptTransaction) => void
+) {
+  const {
+    selection: { $from },
+    tr,
+  } = state
+
+  const rootList = findRootList($from)
+
+  if (rootList) {
+    state.doc.nodesBetween(
+      rootList.pos,
+      rootList.pos + rootList.node.nodeSize,
+      (node, pos) => {
+        const $fromPos = tr.doc.resolve(tr.mapping.map(pos))
+        const $toPos = tr.doc.resolve(tr.mapping.map(pos + node.nodeSize - 1))
+        const nodeRange = $fromPos.blockRange($toPos)
+        if (!nodeRange) {
+          return
+        }
+
+        const targetLiftDepth = liftTarget(nodeRange)
+        if (targetLiftDepth || targetLiftDepth === 0) {
+          tr.lift(nodeRange, targetLiftDepth)
+        }
+      }
+    )
+    dispatch(skipTracking(tr))
```

Comment:
is this temporary or we never want to track that?

Reply:
yes this is temporary, until we support tracking both delete+insert in one change then will remove it